src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php

```diff
@@ -196,8 +196,13 @@ public function testTranslator()
             '->registerTranslatorConfiguration() finds Form translation resources'
         );
         $ref = new \ReflectionClass('Symfony\Component\Security\Core\SecurityContext');
+        $ref = dirname($ref->getFileName());
+        if (!file_exists($ref.'/composer.json')) {
+            $this->assertFalse(file_exists($ref.'/Resources/translations/'));
```

Comment:
This one ensure that we won't merge in Core/Resources/translations again (should be removed on 2.7)